src/modules/organisation/api/organisation.controller.ts

```diff
@@ -329,4 +348,56 @@ export class OrganisationController {
             throw SchulConnexErrorMapper.mapSchulConnexErrorToHttpException(result);
         }
     }
+
+    @Delete(':organisationId/klasse')
+    @HttpCode(HttpStatus.NO_CONTENT)
+    @ApiOperation({ description: 'Delete an organisation of type Klasse by id.' })
```

Comment:
Maybe @ApiNoContentResponse is a better choice? Swagger Docu lists 200, but no description for a 204 response